src/components/ItemsList.js

```diff
@@ -6,7 +6,7 @@ function ItemsList() {
   return (
     <FirestoreCollection
       path="items"
-      sort="added_on:desc"
+      // sort="added_on:desc"  => disable sorting for now
```

Comment:
I love the fact that you have been playing with the collection sorting, that is great!, the only thing is that it's adding a little bit of noise for the reviewer. I mean that we should try to stick to the AC, for example, with git, once we are happy with our PR, we can create a new branch from it to explore and experiment.